articles/concepts/Glossary.md

```diff
@@ -12,31 +12,42 @@ uid: microsoft.quantum.glossary
+|<a id="controlled"></a>Controlled|A quantum [operation](xref:microsoft.quantum.glossary#operation) that takes one or more [qubits](xref:microsoft.quantum.glossary#qubit) as enablers for the target operation. For more information, see [Controlled](xref:microsoft.quantum.language.type-model#controlled).|
+|<a id="dirac"></a>Dirac Notation|A symbolic shorthand that simplifies the representation of [quantum states](xref:microsoft.quantum.glossary#quantum-state), also called *bra-ket* notation.  The *bra* portion represents a row vector, for example  $\bra{A} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix}$ and the *ket* portion represents a column vector, $\ket{B} = \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix}$.  Bras and kets can be combined, for example $\braket{A \| B} = \begin{bmatrix} A{_1} & A{_2} \end{bmatrix} \begin{bmatrix} B{_1} \\\\ B{_2} \end{bmatrix} = A{_1}B{_1} + A{_2}B{_2}$, or can represent the same quantum state in different ways, for example, $\ket{+} = H \ket{0} = \frac{1}{\sqrt{2}} \left(\ket{0} + \ket{1} \right) = \frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}$. For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|
+|<a id="eigenvalue"></a>Eigenvalue|The factor by which the magnitude of an [eigenvector](xref:microsoft.quantum.glossary#eigenvector) of a given transformation is changed by the application of the transformation.  Given a square matrix $M$ and an eigenvector $v$, then $Mv = cv$, where $c$ is the eigenvalue and can be a complex number of any argument. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
+|<a id="eigenvector"></a>Eigenvector|A vector whose direction is unchanged by a given transformation and whose magnitude is changed by a factor corresponding to that vector's [eigenvalue](xref:microsoft.quantum.glossary#eigenvalue). Given a square matrix $M$ and an eigenvalue $c$, then $Mv = cv$, where $v$ is an eigenvector of the matrix and can be a complex number of any argument. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
+|<a id="entanglement"></a>Entanglement|Quantum particles, such as [qubits](xref:microsoft.quantum.glossary#qubit), can be connected or *entangled* such that actions performed on one immediately duplicate on the other. Entanglement is essential to [measuring](xref:microsoft.quantum.glossary#measurement) the [state](xref:microsoft.quantum.glossary#quantum-state) of a qubit.  For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
```

Comment:
This is not true. "immediately duplicate" implies faster-than-light information transfer, which is prohibited. Can we change it to "...entangled such that they cannot be described independently from each other. Their measurement results are correlated even when they are separated infinitely far away."